config.py

```diff
@@ -0,0 +1,12 @@
+# omni
+# TELEGRAM_BOT_TOKEN = "5925005582:AAGmVJKdFfgjajayG2XJUMk2y3iRLPCOVME"
+# shopify
+TELEGRAM_BOT_TOKEN = "5846396253:AAH-zzCV3XuoqAJDFeWg7Po_lh41jT30PMc"
+# free version
+CHAT_GPT_API_KEY = ''
+# paid version
+# CHAT_GPT_API_KEY = ''
+
+import os
+bot_token = os.getenv("TELEGRAM_BOT_TOKEN")
```

Comment:
осылай жасаған дұрыс шығар

```suggestion
bot_token = os.getenv("TELEGRAM_BOT_TOKEN")
if bot_token == "":
     bot_token = TELEGRAM_BOT_TOKEN
```

Reply:
осылай дұрыс болатын шығар, қалай ойлайсың?
```
bot_token = os.getenv("TELEGRAM_BOT_TOKEN")
if bot_token == "":
     bot_token = TELEGRAM_BOT_TOKEN
 ```